pkgs/applications/networking/browsers/firefox/default.nix

```diff
@@ -64,7 +65,6 @@ stdenv.mkDerivation rec {
       "--disable-necko-wifi" # maybe we want to enable this at some point
       "--disable-installer"
       "--disable-updater"
-      "--disable-pulseaudio"
```

Comment:
I've always somewhat lacked in understanding of this (and haven't found definitive sources), so please correct me -- as I've understood, Nix calculates dependencies of an output by grepping all its potential inputs (buildInputs, antiquotes, everything that was mentioned) inside a directory. If so, then Firefox wouldn't pull PulseAudio when installing from a binary source -- I've grepped firefox output and there are no mentions of libpulseaudio path.